pkg/applicationmanager/controllers/applicationbackup.go

```diff
@@ -501,6 +509,12 @@ func (a *ApplicationBackupController) backupVolumes(backup *stork_api.Applicatio
 			if pvc.Status.Phase != v1.ClaimBound || pvc.DeletionTimestamp != nil {
 				continue
 			}
+			// Don't backup PVCs which are already added to Status and for
+			// which backup was triggered
+			if _, isVolBackupDone := backupStatusVolMap[pvc.Namespace+"-"+pvc.Name]; isVolBackupDone {
```

Comment:
backupStatusVolMap is [string]string also I don't see `backupStatusVolMap` getting updated anywhere ?

Reply:
oops.. remnant of some alternate approach I was trying..will remove this snippet